opm/simulators/linalg/bda/opencl/kernels/ILU_apply1_fm.cl

```diff
@@ -14,7 +14,7 @@ __kernel void ILU_apply1(
     const unsigned int block_size,
     __local double *tmp)
 {
-    const unsigned int warpsize = 32;
+    const unsigned int warpsize = 64;
```

Comment:
Seems like for AMD this recommended, whereas INTEL (and NVIDIA) recommends 32,

Should we make this configurable or choose based on hardware vendor? Maybe use `CL_KERNEL_PREFERRED_WORK_GROUP_SIZE_MULTIPLE`?

Reply:
Now implemented